res/code/bml.php

```diff
@@ -106,20 +108,9 @@ function getutf8($s) {
   
   // Read the KeymanWeb code from s.keyman.com/
   $KeymanWebRoot = "{$KeymanCloudRootPath}kmw\\engine\\{$ver}";
-  
-  //
-  // Note: we need at least version 2.0.465
-  //
-  
-  if(intval($build) < 465) {
```

Comment:
This test is no longer required for KeymanWeb 10 as we always use full release number.